bin/analyze_hoc_activity

```diff
@@ -21,12 +21,16 @@ def add_count_to_hash(hash, key, count)
 end
 
 def analyze_day_fast(date)
```

Comment:
I checked and it looks like our systems are all configured for UTC in Rails, the server, and the database, so they're at least consistent, and there shouldn't be any timezone-related issues using the default date-time operations among them.